components/Author/Profile/AuthorHeaderExpertise.tsx

```diff
@@ -17,6 +17,7 @@ const AuthorHeaderExpertise = ({ profile }: { profile: FullAuthorProfile }) => {
       {profile.reputationList.slice(0, REP_GAUGES_TO_SHOW).map((rep, index) => (
         <div className={css(styles.reputation)}>
           <div className={css(styles.reputationHubLabel)}>{rep.hub.name}</div>
+          {/* {rep.percentile} */}
```

Comment:
Nit: What is the purpose of this comment?

Reply:
Temporarily commenting out. Need Tyler/Taki input on this since the values don't feel right. Chatting on slack with them.